keps/sig-storage/1472-storage-capacity-tracking/README.md

```diff
@@ -774,10 +760,10 @@ enhancement:
   Yes.
 
   When disabling support in the API server, the new object type and
-  thus all objects will disappear together with the new flag in
+  will disappear together with the new flag in
   `CSIDriver`, which will then cause kube-scheduler to revert to
   provisioning without capacity information. However, the new objects
-  will still remain in the etcd database.
+  will still remain in the etcd database, they just won't be visible.
```

Comment:
Hmm - so what exactly is happening. Will the CRD itself be deleted?
If so, the underlying CRs will also get deleted, no?

Reply:
Actually - is it CRD or is it a built-in API? I think it's not explicitly mentioned anywhere...

Reply:
Built-in, see https://github.com/kubernetes/enhancements/pull/1829#discussion_r559656546.

Reply:
I've clarified that at the top of the API section.